.github/workflows/docs.yaml

```diff
@@ -0,0 +1,55 @@
+name: Docs Preview
+
+on:
+  pull_request:
+
+jobs:
+  preview_docs:
+    timeout-minutes: 30
+    name: Docs preview
+    if: "github.event_name == 'pull_request'"
+    runs-on: ubuntu-latest
+    env:
+      RUSTC_WRAPPER: "sccache"
+      SCCACHE_GHA_ENABLED: "on"
+      PREVIEW_PATH: pr/${{ github.event.pull_request.number }}/docs
```

Comment:
You also need to set the cache size.  If we have one action that uses sccache with a different size it will blow away half of the cache of the runner every time it runs.  The default size is a lot smaller:
https://github.com/n0-computer/iroh/blob/0fc37942be3d68399fbe45401ba7d67be43a83a6/.github/workflows/ci.yml#L20

Reply:
Oh shit. Thanks! Good call.